libraries/loader/src/loader.js

```diff
@@ -3,6 +3,7 @@ const assume = require('assume');
 const TopoSort = require('topo-sort');
 const debug = require('debug')('taskcluster-lib-loader');
 
+let myLoader;//variable to store method load() so that er can use it globally
```

Comment:
I don't think we need to declare this at module level (globally), since we're only referencing it inside the closure and not exporting it anywhere.